src/puppetlabs/puppetdb/pql/transform.clj

```diff
@@ -2,15 +2,21 @@
   (:require [clojure.string :as str]
             [puppetlabs.puppetdb.cheshire :as json]))
 
+(defn paging-clause?
```

Comment:
I don't think this would be done in instaparse -- we'd have to tag the return values of the transform functions in this ns with metadata. IMO doing this just for paging clauses would be a bit weird, and doing it for all the transform return values would be unnecessary.